packages/esm-patient-vitals-app/src/vitals/vitals-header/vital-header-state.component.tsx

```diff
@@ -22,6 +22,7 @@ interface VitalHeaderProps {
 const VitalHeader: React.FC<VitalHeaderProps> = ({ patientUuid, showRecordVitals }) => {
   const { t } = useTranslation();
   const config = useConfig();
+  const { concepts } = React.useMemo(() => config, [config]);
```

Comment:
I find it odd that we receive a changed config. Should we improve the `useConfig` hook? I think it should directly give us something that won't change between re-renders.